### ISSUE TITLE
Images not being updated

### ISSUE THREAD

**Issue by evanj** [GoogleContainerTools/distroless#452]
Historically, when a commit was merged to master, the images were all published. That no longer appears to be happening. Some commits were merged last week, but the latest images available are from October 28th. E.g. see the gcr.io/distroless/base image:

https://console.cloud.google.com/gcr/images/distroless/GLOBAL/base?gcrImageListsize=30

The latest build there is the following:

```
commit 22bd467b41e5e656e31db347265fae118db166d9
Author: Chanseok Oh <chanseok@google.com>
Date:   Mon Oct 28 13:14:09 2019 -0400

    Upgrade Java and Jetty (#424)
```

This is true for ALL the images I looked at: `python3`, `python2.7`, `static`, etc. I imagine this means some part of the master build on Google's side is broken?

**Comment by evanj** [comment]
I don't know what happened, but as of 6 days ago the images are being published! 🎉 This is now up to date with master as of this writing (commit `5f6acac4bfda8895244a07c2680bbba6e49f0519`)

(I noticed that some images are being tagged with the git commits that built them and some are not, but that is not a new issue so I'm going to ignore that. Thanks!)

**Comment by chanseokoh** [comment]
This is fixed by #454. Should have closed it.
